source/services/index.md

```diff
@@ -0,0 +1,109 @@
+An `Ember.Service` is a long-lived `Ember.Object` available application-wide. Services are singletons; only one instance of a given service can exist at a time.
```

Comment:
"An `Ember.Service` is a long-lived `Ember.Object` available application-wide." is probably still too jargony. It's _true_ but I don't know that it's helpful for someone who doesn't already know this.